{{ cookiecutter.repo_name }}/.circleci/config.yml

```diff
@@ -533,8 +533,8 @@ jobs:
           at: ~/{{ cookiecutter.repo_name }}
 
       - run:
-          name: Deploy to beta track on HockeyApp
-          command: ./gradlew app:uploadProductionReleaseToHockeyApp --info --no-daemon
+          name: Deploy to production track on App Center
+          command: ./gradlew assembleProductionRelease appCenterUploadProductionRelease --info --no-daemon
```

Comment:
I would use incremental build function and make appCenterUploadProductionRelease dependent on assembleProductionRelease, So assembleProductionRelease doesn't need to be specified.

Reply:
Also a good point. I can update the configuration plugin to do this internally.

Reply:
It seems the plugin is providing an additional task which depends on assemble ```project.tasks.register(
                        "appCenterAssembleAndUpload$taskSuffix"
                    ) { task ->
                        task.group = APP_CENTER_PLUGIN_GROUP
                        task.description = "Assemble and upload ${variant.name} APK to AppCenter"
                        task.dependsOn(uploadTask, assembleTask)
                    }```  https://github.com/oliviergauthier/gradle-appcenter-plugin/blob/master/src/main/kotlin/com/betomorrow/gradle/appcenter/AppCenterPlugin.kt. It's weird i don't see it.

Reply:
The other tasks has been added in version 1.1.17 of the plugin.

Reply:
If you update the plugin, you'll get new tasks `appCenterAssembleAndUpload<Variant>`. I don't understand why they just didn't put the dependsOn on the original task instead of creating a new one. I don't see a use case where it could be useful to upload an apk without making sure it was generated first. This was the default behavior of the HockeyApp plugin  we used before.